libs/entry-components/styles/partials/core/components/tables/_action-cells.scss

```diff
@@ -1,16 +1,13 @@
-.entry-table .mat-icon-button {
-	padding-top: 8px;
-	border: none;
-	background-color: transparent;
-	cursor: pointer;
-}
-
 .entry-table-checkbox-cell {
-	flex: none;
-	width: 56px;
-	max-width: 40px;
+	width: 46px;
 
 	.mat-mdc-checkbox {
-		display: flex;
+		.mdc-form-field {
+			margin-bottom: 0;
```

Comment:
Why do we need this margin? I don't think we need this.

Reply:
because in forms/general we have .mdc-form-field with margin-bottom: 10px. Checkbox also has the same class name inside it, it is not aligned with the text in the columns (it goes up 10px)

Reply:
That's for forms. It isn't applied on table since table isn't in form. As you can see...

![image](https://github.com/enigmatry/entry-angular-building-blocks/assets/47369465/a9e19d77-de6a-4b37-b972-db6935a698f8)